src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -62,153 +62,225 @@
-                                                  BorderBrush="{TemplateBinding BorderBrush}"
-                                                  BorderThickness="{TemplateBinding BorderThickness}"
-                                                  AutomationProperties.AccessibilityView="Raw"
-                                                  Content="{TemplateBinding Content}"
-                                                  ContentTemplate="{TemplateBinding ContentTemplate}"
-                                                  ContentTransitions="{TemplateBinding ContentTransitions}"
-                                                  CornerRadius="{TemplateBinding CornerRadius}"/>
+                                <Grid Height="30"
+                                      HorizontalAlignment="Left"
+                                      VerticalAlignment="Top">
+                                    <Grid x:Name="SwitchAreaGrid"
+                                          Margin="0,5"
+                                          Background="{ThemeResource ToggleSwitchContainerBackground}"
+                                          Control.IsTemplateFocusTarget="True"/>
+                                    <Rectangle x:Name="SwitchKnobBounds"
+                                               Fill="{ThemeResource SwitchToggleBackground}"
+                                               RadiusX="4"
+                                               RadiusY="4"/>
+                                    <Grid x:Name="SwitchKnob" HorizontalAlignment="Left">
+                                        <Rectangle Width="32"
+                                                   Fill="{ThemeResource SystemAccentColor}"
+                                                   RadiusX="4"
+                                                   RadiusY="4"/>
+                                        <Grid.RenderTransform>
+                                            <TranslateTransform x:Name="KnobTranslateTransform"/>
+                                        </Grid.RenderTransform>
+                                    </Grid>
+                                    <Rectangle x:Name="OuterBorderStroke"
+                                               Stroke="{ThemeResource SystemAccentColor}"
+                                               StrokeThickness="1"
+                                               IsHitTestVisible="False"
+                                               RadiusX="4"
+                                               RadiusY="4"/>
+                                    <StackPanel x:Name="IconsPanelOff"
+                                                Margin="8,0"
+                                                Orientation="Horizontal">
+                                        <FontIcon x:Name="GraphIcon"
+                                                  Margin="0,0,22,0"
```

Comment:
>22 [](start = 62, length = 2)

There's a little jitter that happens when you toggle.  I think it's because this doesn't line up with the 20 value below.

Reply:
actually line 249 should be 22 for things to line up.

Reply:
Oops, it was a last second changes. Fixed! Thanks!